mw-webapp/src/component/title/Title.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "src/component/title/Title.module.scss";
+
+interface TitleProps {
+  /**
+   * Heading level (h1, h2, h3, h4, h5, h6)
+   */
+  size: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+
+  /**
+   * Text content of the heading
+   */
+  text: string;
+}
+
+export const Title: React.FC<TitleProps> = ({size, text}: TitleProps) => {
+  const HeadingElement = size;
+  return (
+    <HeadingElement
```

Comment:
let's use https://www.radix-ui.com/themes/docs/components/heading as base

Reply:
Can I clarify the question ? Do I need to rename the component "Heading" or I need to add the props (as, size, weight, align, trim, color, contrast)?